packages/design-system-docs/src/pages/startup/migrating-v2.md

```diff
@@ -29,6 +29,34 @@ New usage:
   }
 ```
 
+### Child design system packages (formerly site packages)
+
+The `core`, `support`, `layout` npm packages are no longer needed for applications using a [child design system]({{root}}/startup/child-design-systems/). The only dependency needed is the child design system package in your `package.json`.
```

Comment:
Usage of child design system NPM packages have also been consolidated and simplified. Applications that use child design system packages only need one design system dependency going forward. The child design system packages will contain core styles, components, and assets in addition to child design system specific customizations.